move/its/sources/events.move

```diff
@@ -65,7 +65,11 @@ public(package) fun interchain_transfer<T>(
     amount: u64,
     data: &vector<u8>,
 ) {
-    let data_hash = bytes32::new(address::from_bytes(keccak256(data)));
+    let data_hash = if (data.length() == 0) {
```

Comment:
test event in unit test

Reply:
as part of reviewing tests, can you add event tests as well?

Reply:
added some tests